fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -1898,9 +1912,8 @@ public void computeScanRangeAssignment() throws Exception {
                         minLocation = location;
                     }
                 }
-                Long scanRangeLength = getScanRangeLength(scanRangeLocations.scan_range);
                 assignedBytesPerHost.put(minLocation.server,
-                        assignedBytesPerHost.get(minLocation.server) + scanRangeLength);
+                        assignedBytesPerHost.get(minLocation.server) + 1);
```

Comment:
add some comment why here is 1

Reply:
In old version, scanRangeLength is always 1.